decoders/0005-wazuh_decoders.xml

```diff
@@ -6,7 +6,51 @@
 -->
 
 <decoder name="wazuh">
-  <prematch>^wazuh: </prematch>
+  <prematch>^wazuh: |^ossec: </prematch>
+  <type>wazuh</type>
+</decoder>
+
+<decoder name="wazuh-logcollector">
+  <type>wazuh</type>
+  <prematch>^\d\d\d\d/\d\d/\d\d \d\d:\d\d:\d\d ossec-logcollector|^\d\d\d\d/\d\d/\d\d \d\d:\d\d:\d\d wazuh-logcollector</prematch>
+  <regex offset="after_prematch">^\(\d+\): (\.)</regex>
+  <order>extra_data</order>
+</decoder>
+
+<decoder name="wazuh-syscheck-audit">
+  <parent>wazuh</parent>
+  <type>wazuh</type>
+  <prematch offset="after_parent">^Audit:</prematch>
+  <regex offset="after_prematch">^ (\.+)</regex>
+  <order>extra_data</order>
+</decoder>
+
+<decoder name="wazuh-agent">
+  <parent>wazuh</parent>
+  <type>wazuh</type>
+  <prematch offset="after_parent">^Agent started:</prematch>
+  <regex offset="after_prematch">^ '(\S+\S)'</regex>
+  <order>extra_data</order>
+  <fts>name, location, extra_data</fts>
+</decoder>
+
+<decoder name="wazuh-alert1">
+   <parent>wazuh</parent>
+   <prematch>^ossec: Alert Level:|^wazuh: Alert Level:</prematch>
+   <plugin_decoder>WazuhAlert_Decoder</plugin_decoder>
+</decoder>
+
+<decoder name="wazuh-alert">
+  <program_name>^ossec$|^wazuh$</program_name>
```

Comment:
Probably, this decoder complements the above one. If `wazuh-alert1`, we should analyze if `wazuh-alert` is also only for managers.

Reply:
The **program_name** will remain like this until we can be sure totally sure about this.